server/resources/UserResources.py

```diff
@@ -3,6 +3,7 @@
 from models.UserModel import UserModel
 from models.CampaignModel import CampaignModel
 from flask_jwt_extended import (create_access_token, jwt_required, get_jwt_identity)
+from utils.ValidationDecorator import validate_args
```

Comment:
I think we don't need this anymore, I'll probably remove it in my next pull request